Logger/LogFactory.cs

```diff
@@ -2,10 +2,26 @@
 {
     public class LogFactory
     {
+        private string filePath;
```

Comment:
Fields should be named with underscore PascalCase.
```suggestion
        private string _FilePath;
```